packages/manager/apps/dedicated/client/app/ip/translations/Messages_fr_FR.json

```diff
@@ -453,11 +452,11 @@
   "ip_order_step1_ok": "Cliquez sur \"Suivant\" pour accéder à la commande.",
   "ip_order_add_ipv4_step1_remaining": "Vous avez la possibilité d'ajouter encore <span class=\"blue\">{{t0}}</span> IPv4 sur ce service.",
   "ip_order_step1_info1": "Votre serveur peut supporter jusqu'à :",
-  "ip_order_step1_DEDICATED_pre": "<span class=\"blue\">{{t0}}</span> IP de blocs failover IPv4",
+  "ip_order_step1_DEDICATED_pre": "<span class=\"blue\">{{t0}}</span> IP de blocs Additional IP",
   "ip_order_step1_DEDICATED_v2_pre": "<span class=\"blue\">{{t0}}</span> préfixes",
-  "ip_order_step1_DEDICATED_individual_1": "<span class=\"blue\">1</span> IP failover individuelle",
+  "ip_order_step1_DEDICATED_individual_1": "<span class=\"blue\">1</span> IP additionnelles individuelle",
   "ip_order_step1_DEDICATED_v2_individual_1": "<span class=\"blue\">1</span> préfixe",
-  "ip_order_step1_DEDICATED_individual_other": "<span class=\"blue\">{}</span> IP failover individuelles",
+  "ip_order_step1_DEDICATED_individual_other": "<span class=\"blue\">{}</span> Additional IP individuelles",
```

Comment:
Isn't there something missing in the { }?
Best to remove the html code from the translations.

Reply:
Hello Guillaume, They are using this in order to build a custom filter by framing a html. We cannot remove it. And { } is needed as a text.